spec/controllers/idv/address_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe Idv::AddressController do
```

Comment:
We didn't have a controller spec for this previously, so I added an extremely minimal one.